addons/binding/org.openhab.binding.solarlog/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solarlog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="datalogger">
+		<label>SolarLog Binding Thing</label>
```

Comment:
Isn't there any more suitable label for a "datalogger"?

Reply:
I will rename it to `meter` which should be more appropriate (and in the context of the thing ID will result in `solar log:meter:pv` which should be self-explanatory. Hope that's OK with you.)